doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,57 @@
+.. _software-update-server:
+
+Software Update: Building an Ostro |trade| OS  Repository
+#########################################################
+
+This technical note provides instructions supporting the creation
+of a repository for feeding updates to devices based on Ostro OS.
+
```

Comment:
It was already mentioned explicitly: 
"The information produced by the server is then provided by an HTTP server and consumed by the software update client running on supported devices."

In the new version I've mentioned it twice, also in a note.

Of course 2, 3 or more times doesn't make a difference if the document is not read.